src/mmw/js/src/modeling/models.js

```diff
@@ -130,12 +164,32 @@ var ScenarioModel = Backbone.Model.extend({
         name: '',
         is_current_conditions: false,
         modifications: null,      // ModificationsCollection
-        active: false
+        active: false,
+        uid: null
```

Comment:
If we associated a user ID with a scenario in the DB the column would probably be `user_id` and not `uid` that's my rationale.